person.rb

```diff
@@ -9,6 +9,12 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    private
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
   end
```

Comment:
- It would also be better if the `Person` class has a method for adding rentals. The `person` object itself should be sent as a parameter to create the new rental thus achieving the `has-many` association. Please see the example code below for an example of how to achieve this.

## Example code
```ruby
require_relative('./nameable')
 class Person < Nameable
  attr_reader :id
  attr_accessor :name, :age, :rentals

  def initialize(age, name = 'Unknown', parent_permission: true)
    super()
    @id = Random.rand(1..255)
    @name = name
    @age = age
    @rentals = []
    @parent_permission = parent_permission
  end

  def correct_name
    @name
  end

  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end
end
```

## Project requirements
![Screenshot from 2023-05-25 01-10-32](https://github.com/BahirHakimy/OOP-school-library/assets/16392046/f47a0e12-dbe2-4ccf-a44c-95426147e7f1)

Reply:
Thanks, @Felix45 for the review
I will add all suggested changes